client/dashboard/store-performance/index.js

```diff
@@ -262,19 +262,13 @@ export default compose(
 
 		const primaryQuery = {
 			after: appendTimestamp( datesFromQuery.primary.after, 'start' ),
-			before: appendTimestamp(
```

Comment:
Why was this changed? How does this change behaviour?

Reply:
I changed this because it essentially made the request for performance indicators impossible to cache. Without the change, if your chosen date range ended on "today", the `before` timestamp became the value "now" - down to the second.

Since we're already [busting cache on sync](https://github.com/woocommerce/woocommerce-admin/search?q=invalidate&unscoped_q=invalidate) I decided to force this query to always be to the end of day (`T23:59:59`) so that the request would be the same, which is what the cache key is built off of.

Reply:
Ah of course, that makes sense. Thanks.